qiita_db/worker.py

```diff
@@ -0,0 +1,7 @@
+import sys
```

Comment:
This should be in the scripts folder so it's added automatically or in the qiita_db/test/support_files/